src/pybind/mgr/nfs/export.py

```diff
@@ -543,7 +552,7 @@ def _change_export(self, cluster_id: str, export: Dict) -> Dict[str, str]:
         except Exception as ex:
             msg = f'Failed to apply export: {ex}'
             log.exception(msg)
-            return {"state": "error", "msg": msg}
+            return {"state": "error", "msg": msg, "exception": ex}
```

Comment:
What's the need to have a separate `exception` field?

Reply:
this will be used to return the err on CLI. Dont worry this wont be part of the output JSON, this gets removed(`pop()`) before sending the results on CLI

Reply:
In that case, what's the need to include in the json when it can be maintained as a class variable?